File: content/code-examples/tested/java/driver-sync/timeseries/AutoRemoval.snippet.get-timeseries-collection-expiry.java
Document filter = new Document("name", "weather24h");
Document collectionInfo = database.listCollections().filter(filter).first();
if (collectionInfo != null) {
    Document options = collectionInfo.get("options", Document.class);
    if (options != null) {
        return options.getLong("expireAfterSeconds");
    }
}

File: content/code-examples/tested/csharp/driver/TimeSeries/AutoRemoval.snippet.get-timeseries-collection-expiry.cs
var collectionInfoCursor = await
    database.ListCollectionsAsync(
        new ListCollectionsOptions { Filter = new BsonDocument("name", "weather24h") });
var collectionInfo = await collectionInfoCursor.FirstOrDefaultAsync();
if (collectionInfo != null)
{
    return collectionInfo["options"]["expireAfterSeconds"];
}
